### PR TITLE
Add support for testing format output functions (fixes #2459)

### LINKED ISSUE #2459: Enable placeholder count inspection for formatted testing functions as well
The plug should highlight the placeholder count issue for testing methods `Logf`, `Skipf`, `Errorf` and `Fatalf`.

### PR DESCRIPTION
**Pull request by dlsniper**

### CODE REVIEW

File: src/com/goide/inspections/GoPlaceholderCountInspection.java [hunk 1]
```diff
@@ -49,13 +49,15 @@
...
+    Pair.pair("panicf", 0),
+    Pair.pair("logf", 0),
+    Pair.pair("skipf", 0));
 
   private static int getPlaceholderPosition(@NotNull GoFunctionOrMethodDeclaration function) {
     Integer position = FORMATTING_FUNCTIONS.get(StringUtil.toLowerCase(function.getName()));
     if (position != null) {
       String importPath = function.getContainingFile().getImportPath(false);
-      if ("fmt".equals(importPath) || "log".equals(importPath)) {
+      if ("fmt".equals(importPath) || "log".equals(importPath) || "testing".equals(importPath)) {
```
Comment (zolotov) [inline src/com/goide/inspections/GoPlaceholderCountInspection.java]: com.goide.GoConstants#TESTING_PATH

Comment (dlsniper) [inline src/com/goide/inspections/GoPlaceholderCountInspection.java]: Done, thanks.

**Comment by zolotov** [comment]
How about reimplement this eventually? You can use govet test data: https://github.com/golang/go/blob/master/src/cmd/vet/testdata/print.go

**Comment by dlsniper** [comment]
I'd rather try and understand how to do the testing / building / debugging first then do a reimplementation for this.

**Comment by zolotov** [comment]
Well, as you with but do not forget that the difference between these issues that the inspection is already provided buggy implementation and tests debugging is totally new feature

**Pull request merged**